src/installer.ts

```diff
@@ -170,20 +192,65 @@ export async function getNode(
   core.addPath(toolPath);
 }
 
+function isLtsAlias(versionSpec: string): boolean {
+  return versionSpec.startsWith('lts');
+}
+
+function getManifest(auth: string | undefined): Promise<tc.IToolRelease[]> {
+  core.debug('Getting manifest from actions/node-versions@main');
+  return tc.getManifestFromRepo('actions', 'node-versions', auth, 'main');
+}
+
+function resolveLtsAliasFromManifest(
+  versionSpec: string,
+  stable: boolean,
+  manifest: INodeRelease[]
+): string {
+  const alias = versionSpec.split('lts/')[1]?.toLowerCase();
+
+  if (!alias) {
+    throw new Error(
+      `Unexpected LTS alias '${alias}' for Node version '${versionSpec}'`
```

Comment:
Empty quotes in the error message looks a bit strange to me, maybe we should re-phrase to something like `LTS alias is empty for Node version '${versionSpec}'`

Reply:
Good catch, changed to something more generic.

Reply:
@AlenaSviridenko Please let me know if that looks good to you. I decided to not be specific about alias being **empty**, but rather throw generic parse error.

Reply:
I like how it looks now 🙂